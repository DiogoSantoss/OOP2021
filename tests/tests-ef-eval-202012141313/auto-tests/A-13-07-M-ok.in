6
3
C2
C2C2
C2C2C2
0
5
5
M2
1
1
0
6
2
1
C2
1
C1
0
0
