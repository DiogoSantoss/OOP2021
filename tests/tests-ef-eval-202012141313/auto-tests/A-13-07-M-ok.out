Menu Principal
1 - Abrir
2 - Guardar
3 - Mostrar Data
4 - Avançar Data
5 - Gestão de Produtos
6 - Gestão de Clientes
7 - Gestão de Fornecedores
8 - Gestão de Transacções
9 - Menu de Consultas
10 - Ver Saldo Global
0 - Sair
Escolha uma opção: Gestão de Clientes
1 - Mostrar cliente
2 - Mostrar clientes
3 - Registar cliente
4 - Activa/desactivar notificações de um produto
5 - Mostrar transacções do cliente
0 - Sair
Escolha uma opção: Identificador do cliente: Nome do cliente: Endereço do cliente: Gestão de Clientes
1 - Mostrar cliente
2 - Mostrar clientes
3 - Registar cliente
4 - Activa/desactivar notificações de um produto
5 - Mostrar transacções do cliente
0 - Sair
Escolha uma opção: Menu Principal
1 - Abrir
2 - Guardar
3 - Mostrar Data
4 - Avançar Data
5 - Gestão de Produtos
6 - Gestão de Clientes
7 - Gestão de Fornecedores
8 - Gestão de Transacções
9 - Menu de Consultas
10 - Ver Saldo Global
0 - Sair
Escolha uma opção: Gestão de Produtos
1 - Visualizar
2 - Registar Caixa
3 - Registar Contentor
4 - Registar Livro
5 - Alterar Preço
0 - Sair
Escolha uma opção: Identificador do produto: Preço: Gestão de Produtos
1 - Visualizar
2 - Registar Caixa
3 - Registar Contentor
4 - Registar Livro
5 - Alterar Preço
0 - Sair
Escolha uma opção: BOX|C6H5OH|S1|2|20|100|NORMAL
BOX|H2|S1|4|20|100|AIR
CONTAINER|M2|S1|1|20|100|AIR|C4
CONTAINER|M3|S1|16|20|0|PERSONAL|DL
BOOK|PB1256|S1|58|2|5|Os Lusíadas|Luís de Camões|1234567890
BOOK|PB9854|S1|10|2|5|Head First Java|Sierra & Bates|98765432
Gestão de Produtos
1 - Visualizar
2 - Registar Caixa
3 - Registar Contentor
4 - Registar Livro
5 - Alterar Preço
0 - Sair
Escolha uma opção: Menu Principal
1 - Abrir
2 - Guardar
3 - Mostrar Data
4 - Avançar Data
5 - Gestão de Produtos
6 - Gestão de Clientes
7 - Gestão de Fornecedores
8 - Gestão de Transacções
9 - Menu de Consultas
10 - Ver Saldo Global
0 - Sair
Escolha uma opção: Gestão de Clientes
1 - Mostrar cliente
2 - Mostrar clientes
3 - Registar cliente
4 - Activa/desactivar notificações de um produto
5 - Mostrar transacções do cliente
0 - Sair
Escolha uma opção: C1|R2-D2|somewhere|NORMAL|0|0
C2|C2C2|C2C2C2|NORMAL|0|0
Gestão de Clientes
1 - Mostrar cliente
2 - Mostrar clientes
3 - Registar cliente
4 - Activa/desactivar notificações de um produto
5 - Mostrar transacções do cliente
0 - Sair
Escolha uma opção: Identificador do cliente: C2|C2C2|C2C2C2|NORMAL|0|0
BARGAIN|M2|1
Gestão de Clientes
1 - Mostrar cliente
2 - Mostrar clientes
3 - Registar cliente
4 - Activa/desactivar notificações de um produto
5 - Mostrar transacções do cliente
0 - Sair
Escolha uma opção: Identificador do cliente: C1|R2-D2|somewhere|NORMAL|0|0
BARGAIN|M2|1
Gestão de Clientes
1 - Mostrar cliente
2 - Mostrar clientes
3 - Registar cliente
4 - Activa/desactivar notificações de um produto
5 - Mostrar transacções do cliente
0 - Sair
Escolha uma opção: Menu Principal
1 - Abrir
2 - Guardar
3 - Mostrar Data
4 - Avançar Data
5 - Gestão de Produtos
6 - Gestão de Clientes
7 - Gestão de Fornecedores
8 - Gestão de Transacções
9 - Menu de Consultas
10 - Ver Saldo Global
0 - Sair
Escolha uma opção: 